src/main/java/uwu/command/DeleteCommand.java

```diff
@@ -0,0 +1,61 @@
+package uwu.command;
+
+import uwu.exception.NullTaskException;
+import uwu.exception.UwuException;
+
+import uwu.Storage;
+
+import uwu.task.Task;
+import uwu.task.TaskList;
+
+import uwu.Ui;
+
+/**
+ * Deletes a task from the task list.
+ */
+public class DeleteCommand extends Command {
+    /** The index of the task to be deleted. */
+    int index;
```

Comment:
I noticed that in some classes, such as Uwu bot, access modifiers were indicated for variables, maybe you might want to consider standardizing that for all the classes?